front/components/assistant/RenderMessageMarkdown.tsx

```diff
@@ -216,14 +239,32 @@ export function RenderMessageMarkdown({
       mention: ({ agentName }) => {
         return <MentionBlock agentName={agentName} />;
       },
+      // @ts-expect-error - `mention` is a custom tag, currently refused by
+      // react-markdown types although the functionality is supported
+      visualization: ({ position }) => {
+        return (
+          <VisualizationActionIframe
+            owner={owner}
```

Comment:
Kind of sad that this generic component becomes workspace aware.

Reply:
Your design that viz should be workspace specific :)

Reply:
Required to fetch the files 🙃.